### ISSUE TITLE
Copy from other profile don't copy some setting

### ISSUE THREAD

**Issue by Vucumpra** [osmandapp/OsmAnd#19767]
### Description

Hello, copying setting from another profile may not be done completely

### Steps to reproduce

- Have minimum 2 profile on your osmand, **A** and **B**
- **B** profile should have "Custom button" and "Top panel" personalised.
- Top panel should have widget with "minimal" size.

- Open **A** profile.
- Go to configure_profile >configure_screen
- In bottom screen : select Copy from another profile
- Select **B** profile and push "Copy" button.
- Escape from menu and stay on **A** profile.

### Actual result

- Now you can see that all setting is not copied : 
For exemple, personalised "Custom Buttons" are not present and Widget in top bar back with "medium" size.

### Expected result

A complete copy of selected element

### Your Environment (required)

**WARNING** Crash-Logs **MAY** contain information you deem sensitive.
Review this **CAREFULLY** before posting your issue!

```
OsmAnd Version: F-droid 4.7.10
Android/iOS version: Android 13
Device model: Xiaomi Redmi m2101k7bny
Crash-Logs: ?
```

**Comment by yuriiurshuliak** [comment]
The bug was partially reproduced. I experimented with adding a custom button and small widgets to a cycling profile. After that, I tried copying these settings from the bike profile to the car profile.
It appears that the settings were successfully transferred, including the custom button. However, I observed that although the widgets were copied, they were not the small size as intended but instead appeared as medium-sized widgets.

OsmAnd~ 4.8.0#2277m, released: 2024-05-09

https://github.com/osmandapp/OsmAnd/assets/127092082/072c2b5f-4658-4e33-a682-33f843f0aab2
